src/openrpc/localization.json

```diff
@@ -274,7 +274,105 @@
           }
         }            
       ]   
-    } 
+    },
+    {
+      "name": "addAdditionalInfo",
+      "tags": [
+        {
+          "name": "capabilities",
+          "x-manages": [
+            "xrn:firebolt:capability:localization:language"
```

Comment:
let's make a new capability for this:
`xrn:firebolt:capability:localization:additional-info`
Also needs to change on the `removeAdditionalInfo` and `additionalInfo` methods.